protocol/x/listing/keeper/grpc_query_markets_hard_cap_test.go

```diff
@@ -0,0 +1,42 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/dydxprotocol/v4-chain/protocol/x/listing/types"
+	"github.com/stretchr/testify/require"
+
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+)
+
+func TestQueryPMLEnabledStatus(t *testing.T) {
+	tests := map[string]struct {
+		hardCap uint32
+	}{
+		"Hard cap - 0": {
```

Comment:
nit: `Hard cap is 0` or `Hard cap: 0`, I read this as `Hard cap minus 0` at first.